### ISSUE TITLE
SSL/HTTPS conteúdo inseguro - Firefox bloqueando conteúdo do OSC

### ISSUE THREAD

**Issue by davirs** [deivisonarthur/OSC-Magento-Brasil-6-Pro#106]
Olá.
A princípio eu achei que fosse um problema do meu  tema, mas eu instalei no magento padrão e deu o mesmo problema.

Quando carrega a página do checkout no firefix, a barra se segurança não fica verde como deveria e diz "O firefox está bloqueando conteúdo nesta página".. vou colocar a imagem para vcs analizarem.

Alguém já percebeu isso ou tem ideia do que possa ser?

![errohttps](https://cloud.githubusercontent.com/assets/4355062/7274907/1826c208-e8d7-11e4-90e9-b3cb57416e8f.png)

**Comment by rafaelstz** [comment]
Sim, tem algum conteúdo que provavelmente seja imagem carregando através de um http simples.
**A solução**: Trocar a url de inserção para https, como a URL.
